Tests/Components Tests/IssuerList/IssuerListComponentTests.swift

```diff
@@ -54,6 +61,29 @@ class IssuerListComponentTests: XCTestCase {
 
         waitForExpectations(timeout: 10)
     }
+    
+    func testSearch() {
```

Comment:
This name doesn't explain what we actually test.
The better way to name it would be something like `test_componentSendsInfo_onSearchInput`
It also feels we are not testing real events we send, only count. This is essential part of the business logic of this scope so I suggest to check for real data that we send.

Reply:
6bf6afab0029fd741e69b4a44a732cb546211610 Updated name but hard to test the actual event itself via this way. But current test still checks on the right flow and feels enough to me. We can add more later!